### ISSUE TITLE
Add an option to remove the help button

### ISSUE THREAD

**Issue by evur** [go-gitea/gitea#24714]
### Feature Description

It would be nice to have an option to disable rendering the help button in the navbar.

**Comment by lunny** [comment]
You can customize your template page to do that.

**Comment by evur** [comment]
I did that already. But each template that is overwritten must also be kept up to date.
This also defeats the purpose of having an `extra_links.tmpl`.
So having an option for it would be a much better.

**Comment by lunny** [comment]
Or maybe we should move the mean to a standalone template.

**Comment by wxiaoguang** [comment]
I guess that Help doesn't make sense to end users, that's more like Gitea's manual , more related to configuration , but not helping about the Gitea instances.

So I think it could be moved into admin page?

**Comment by lunny** [comment]
> I guess that Help doesn't make sense to end users, that's more like Gitea's manual , more related to configuration , but not helping about the Gitea instances.
> 
> So I think it could be moved into admin page?

No. The latest documentation contains both configuration and end user documentations.

**Comment by evur** [comment]
> No. The latest documentation contains both configuration and end user documentations.

Maybe it would be good to separate the end user documentation.
The help link could also be moved into the footer.

**Comment by lunny** [comment]
The link is to the help documentation's home page. Although most people are end users, but administrators will visit the documentation more frequently.

**Comment by evur** [comment]
When I want to access Gitea's admin docs, I just type the topic into my browser's address bar and either get it from my history or search it. So I think the installation, admin, etc. sections don't justify a help button.

But the help button could point to a new landing page of the separated user docs.
There could also be a button on this landing page that leads you to the admin docs.

Besides that, little help buttons would be a great addition.

Mockup for secrets:
![](https://github.com/go-gitea/gitea/assets/63237520/3f2f2bbf-3d8e-4b2c-91c7-a7abb8ba3c03)
![](https://github.com/go-gitea/gitea/assets/63237520/9e76c907-ac40-4cfa-861f-29e7a73f477d)
